ethylene-core/src/main/java/com/github/steanky/ethylene/core/bridge/ConfigBridges.java

```diff
@@ -92,25 +92,24 @@ private static void writeInternal(OutputStream outputStream,
     }
 
     /**
-     * <p>Produces a ConfigBridge implementation capable of reading and writing to the given file, using
-     * {@link FileInputStream} and {@link FileOutputStream} objects.</p>
+     * <p>Produces a ConfigBridge implementation capable of reading and writing to the given file path.</p>
      *
      * <p>If the file is invalid or cannot be read from, {@link IOException}s will be thrown when attempts are made to
      * read objects from the ConfigBridge.</p>
      *
      * <p>This method uses {@link ConfigBridges#fromStreams(Callable, Callable, ConfigCodec)} to produce its
      * ConfigBridge implementation.</p>
```

Comment:
is noting this necessary

Reply:
regardless of if it's necessary, it's technically incorrect so i'll remove it